spec.html

```diff
@@ -26234,8 +26231,7 @@
           1. If no arguments were passed to this function invocation, let _s_ be `""`.
           1. Else,
             1. If NewTarget is *undefined* and Type(_value_) is Symbol, return SymbolDescriptiveString(_value_).
-            1. Let _s_ be ToString(_value_).
-          1. ReturnIfAbrupt(_s_).
+            1. Let _s_ be ? ToString(_value_).
```

Comment:
Oh never mind the comment above, `new String(Symbol())` will throw. So the RIA should stay.